### ISSUE TITLE
Debugging with webstorm doesn't stop of breakpoints

### ISSUE THREAD

**Issue by iangregsondev** [angular-fullstack/generator-angular-fullstack#27]
Hi there,

Great generator! I found a small problem, it may not be associated with the generator but according to stackexchange I have setup the correct way of debugging a node.js controlled by grunt. I thought I may put my findings here, maybe somebody else is experiencing same issue or has a fix, or is it a known problem ?

I don't need debugging when developing angularjs but i have found i do need it when developing the backend part (express.js / node.js)

I am lost :-)

I setup debugging via this

http://stackoverflow.com/questions/17043484/grunt-debugging-from-webstorm?rq=1

Here is the output from webstorm, it takes a while for the browser to launch but eventually it launches and everything seems ok and I am able to view the site here http://localhost:9000/

But i have added breakpoints to controllers, server.js etc and nothing is hit.

Its a little suspect as I get "ReferenceError: Process is not defined" below but running from the the command line direct doesn't cause this problem.

"C:\Program Files\nodejs\node.exe" --debug-brk=61951 C:\Users\gregsoni\AppData\Roaming\npm\node_modules\grunt-cli\bin\grunt serve
debugger listening on port 61951
[ReferenceError: process is not defined]
Running "serve" task
[ReferenceError: process is not defined]
[ReferenceError: process is not defined]

Running "clean:server" (clean) task
Cleaning .tmp...OK

Running "concurrent:server" (concurrent) task
[ReferenceError: process is not defined]
[ReferenceError: process is not defined]
[ReferenceError: process is not defined]
[ReferenceError: process is not defined]
[ReferenceError: process is not defined]
[ReferenceError: process is not defined]

Running "copy:styles" (copy) task

Done, without errors.

Running "compass:server" (compass) task
directory .tmp/styles/ 
       create .tmp/styles/main.css (1.237s)
    Compilation took 1.239s

Done, without errors.

Running "autoprefixer:dist" (autoprefixer) task
Prefixed file ".tmp/styles/main.css" created.

Running "express:dev" (express) task
Starting background Express server
connect.multipart() will be removed in connect 3.0
visit https://github.com/senchalabs/connect/wiki/Connect-3.0 for alternatives
connect.limit() will be removed in connect 3.0
Express server listening on port 9000 in development mode

Running "open:server" (open) task
Successfully connected to: mongodb://localhost/test
finished populating things

Running "watch" task
Waiting...GET / 304 10ms
GET /partials/main 304 2ms
GET /api/awesomeThings 200 8ms - 1.03kb

**Comment by qt911025** [comment]
My way is delete the sub-target "concurrent:debug" of task "serve:debug"(I made another target named "debug-pre").
Then I just configured WS debug (same to spy-js) as usual, and add the grunt task("serve:debug-pre") above to "External tool".
I use "Settings"->"Build, Execution, Deployment"-> "Debugger" -> "Live Edit" instead of running nodemon.

I'm so confused before understanding Grunt and tasks in Gruntfile.
